titiler/pgstac/main.py

```diff
@@ -238,3 +245,287 @@ def landing(request: Request):
             "urlparams": str(request.url.query),
         },
     )
+
+
+###############################################################################
+# Register Mosaic
+@app.post(
+    "/mosaic/register",
+    responses={200: {"description": "Register a Search."}},
+    response_model=model.RegisterResponse,
+    response_model_exclude_none=True,
+    tags=["Mosaic"],
+)
+def register_search(
```

Comment:
we move the register/list endpoint outside the `Factor` because it now used `/mosaic/{search_id}` prefix